tfhe/src/high_level_api/keys/public.rs

```diff
@@ -88,7 +103,8 @@ pub struct CompressedPublicKey {
     #[cfg(feature = "shortint")]
     pub(crate) shortint_key: ShortIntCompressedPublicKey,
     #[cfg(feature = "integer")]
-    pub(crate) integer_key: IntegerCompressedPublicKey,
+    pub(in crate::high_level_api) base_integer_key:
+        Option<crate::high_level_api::integers::CompressedPublicKeyDyn>,
```

Comment:
I think I asked elsewhere, but still unclear for now why we have an option here ?